### ISSUE TITLE
T4 Formato de Entrega Reinforcement Learning

### ISSUE THREAD

**Issue by mpspanakakis** [IIC2613-Inteligencia-Artificial-2022-2/Syllabus#88]
Hola, tengo una duda:
En el enunciado se pide un Notebook como entrega del primer parte de la tarea. Para el segundo parte, solo se pide "completar el código". Donde debemos responder las preguntas teóricas de esta parte? En un PDF o Notebook adicional?

Saludos!

**Comment by dfloreaa** [comment]
Efectivamente, las preguntas deben ser respondidas en un pdf adicional a la entrega, lo añadiremos al enunciado
